snafu/dns_perf_wrapper/Readme.md

```diff
@@ -0,0 +1,39 @@
+## DNS Performance Workload Wrapper
+Wrapperf around [dnsperf](https://github.com/DNS-OARC/dnsperf) to measure throughput ( requests per second ) and latency metrics for Domain Name Service ( DNS ). The collected metrics are scraped to generate a json which can be indexed into Elasticsearch datastore.
+
+### Dependencies
+This wrapper requires a couple of dependencies as well as enabling copr repo to download the latest dnsperf package. For more details [see[(https://github.com/DNS-OARC/dnsperf/blob/master/README.md). It's recommended to build and run the [containerized version](Dockerfile) to avoid managing the installation on the host.
+
+### Metrics
+The wrapper runs dnsperf and generates a json document with the results as well as the input configuration/parameters. Benchmark wrapper also wraps around metadata to be indexed into Elasticsearch as explained [here](https://github.com/cloud-bulldozer/benchmark-wrapper#how-do-i-post-results-to-elasticsearch-from-my-wrapper). Here is a sample results document:
+
+```
+{
+    "_index": "snafu-dns_perf-results",
+    "_op_type": "create",
+    "_source": {
+        "uuid": "0f038d28-ef3a-42c0-aea3-435d19686ab1",
+        "cluster_name": "mycluster",
+        "clients": 1,
+        "timestamp": "2021-04-29T19:17:47",
+        "elapsed_time": 6.0132739543914795,
+        "latency_stddev": "0.003210",
+        "avg_latency": "0.012043",
+        "QPS": "19.999700",
+        "runtime": "6.000090",
+        "avg_request_packet_size": "32",
+        "avg_response_packet_size": "60",
+        "queries_lost": "0",
+        "queries_lost_percentage": "0.00%",
+        "queries_completed": "120",
+        "queries_completed_percentage": "100.00%",
```

Comment:
Removing the `%`, the key states it is percentage.

Reply:
Fixed it, thanks.